plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/DigestExtractor.java

```diff
@@ -33,7 +34,8 @@
  *
  * @since 3.0
  */
-class DigestExtractor
+@ThreadSafe
```

Comment:
Any particular reason for adding this annotation to various classes in this PR? We have a number of utility classes with static methods which are implicitly thread-safe as the class has no state, but we don't explicitly document them as such at the moment